web-server/v0.4/src/services/global.js

```diff
@@ -1,8 +1,20 @@
 import request from '../utils/request';
 
 export async function queryDatastoreConfig() {
-  // Note that window.location.pathname should have a trailing slash already.
-  return request('http://localhost:8000/dev.config.json', {
+  // Note that window.location.pathname should already have a trailing slash.
+  let configEndpoint = '';
+  let environment = process.env || 'development';
+  
+  switch(environment) {
+    case 'development':
+      configEndpoint = 'http://localhost:8000/dev.config.json'
+      break
+    case 'production':
```

Comment:
Should we be using a simple `if` statement here instead of a `switch` statement?  What happens if somebody sets `process.env` to "`staging`"?

Reply:
Could you add a comment there to that effect?

But the work "production" does not sound like it fits and seems confusing.  Why not just have one variable called, "`configEndpoint`"?  What does the current approach let you do different?

Reply:
Great, that helps.  Thanks.